e2e/scripts/jira-client.ts

```
@@ -102,28 +102,40 @@ export class JiraE2EClient {
+        console.log(`  Note: Software project failed, trying business type`);
+        // Fall back to business type (no fixVersions on create screen)
+        return this.request<JiraProject>('/rest/api/2/project', {
+          method: 'POST',
+          body: JSON.stringify({
+            key,
+            name,
+            projectTypeKey: 'business',
+            lead,
+          }),
+        });
+      }
```

ESLint

[error] Handle this exception or don't catch it at all.

(sonarjs/no-ignored-exceptions)